lxd/storage/backend_lxd.go

```diff
@@ -5824,6 +5947,9 @@ func (b *lxdBackend) CreateCustomVolumeSnapshot(projectName, volName string, new
 
 	defer unlock()
 
+	// Set the parent volume's UUID.
+	vol.SetParentUUID(parentUUID)
```

Comment:
Any reason you put this here and not immediately other initialising `vol` a few lines higher (to keep the volume initialisation together)?